schemas/witnet/witnet.proto

```diff
@@ -299,6 +296,7 @@ message ConsensusConstants {
     uint64 collateral_minimum = 13;
     uint32 collateral_age = 14;
     uint32 superblock_period = 15;
+    uint32 extra_rounds = 16;
```

Comment:
Would it be too bad to have two separate keys, `extra_commit_rounds` and `extra_reveal_rounds`?

Reply:
we dont find a strong reason to have different numbers for that, so thats the reason why we decide to use only one field

Reply:
Similarly, I don't see any reason to use only one field when we could use two fields, and try different combinations to find out what works better.

Reply:
Oks, we need more opinions to broke the tie :P. What do you think @aesedepece, @mariocao?

Reply:
__Tomayto, tomahto.__